### PR TITLE
Align TOCs and render headings no longer as links

### PR DESCRIPTION
**Pull request by Bukama**
Several pages were rendered with broken TOCs or where headings were rendered as links.
This PR fixes (hopefully) all of them.

### CODE REVIEW

**Comment by Bukama** [comment]
I updated the `pom` file to adopt the outstanding change mentioned and discussed in #445, so we can close this PR together with this here (if the formatting is okay for you ofc).

**Comment by slawekjaranowski** [comment]
> I updated the `pom` file to adopt the outstanding change mentioned and discussed in #445, so we can close this PR together with this here (if the formatting is okay for you ofc).

Do you want to not merge this?

**Review by mthmulders — APPROVED** [review]

**Comment by Bukama** [comment]
> > I updated the `pom` file to adopt the outstanding change mentioned and discussed in #445, so we can close this PR together with this here (if the formatting is okay for you ofc).
> 
> Do you want to not merge this?

Not sure if I understand you correctly, but I want to have the suggested change of #445, that's why I integrated it into this PR (where I massively updated the file anyway).

File: content/apt/guides/index.apt.vm [hunk 1]
```diff
@@ -43,7 +43,7 @@ Documentation
 
  * {{{./introduction/introduction-to-profiles.html}Profiles}}
 
- * {{{./introduction/introduction-to-repositories.html}Repositories}}
+ * {{{.guides//introduction/introduction-to-repositories.html}Repositories}}
```
Comment (slawekjaranowski) [inline content/apt/guides/index.apt.vm:46]: now generate as ...:

```
<li><a href="#a.guides.2F.2Fintroduction.2Fintroduction-to-repositories.html">Repositories</a></li>
```

```suggestion
 * {{{.guides/introduction/introduction-to-repositories.html}Repositories}}
```

Comment (Bukama) [reply content/apt/guides/index.apt.vm]: Good spot! Changed

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**